inference-engine/src/mkldnn_plugin/nodes/mkldnn_generic_node.cpp

```diff
@@ -77,7 +77,7 @@ bool MKLDNNGenericNode::created(const MKLDNNExtensionManager::Ptr &extMgr) {
             extFactory = extMgr->CreateExtensionFactory(ngraphOp);
 
             if (!extFactory)
-                THROW_IE_EXCEPTION_WITH_STATUS(NOT_IMPLEMENTED) << "Descriptor for generic primitive doesn't exist";
+                THROW_IE_EXCEPTION_WITH_STATUS(NOT_IMPLEMENTED);
```

Comment:
Minor comment
Maybe we should add some error message like: ```Cant't create factory for ngraph operation with type: " + ngraphOp->get_type_name() + " and name: " + ngraphOp->get_friendly_name()``` ?

Reply:
I removed the message intentionally because I accumulate all errors inside MKLDNNNode::NodesFactory::create (`errorMessage` variable). In case we cannot support such operation error message will contain string: `[NOT_IMPLEMENTED] Unsupported operation of type: Gather name: Gather_4121` (see PR description). I don't want overburden resulting message with in fact useless information.